A32NX/html_ui/Pages/A32NX_Core/A32NX_BrakeTemp.js

```diff
@@ -0,0 +1,60 @@
+const WHEEL_RPM_THRESHOLD = 10;
+const HEAT_UP_FACTOR = 5.25;
+const COOL_DOWN_FACTOR = 1.15;
+
+class A32NX_BrakeTemp {
+
+    tempCoefficientFunction(brakePosition) {
+        return ((brakePosition / 32767) * HEAT_UP_FACTOR) ** 2;
```

Comment:
`return ((brakePosition / 32767) * HEAT_UP_FACTOR) ** 2;` should be `return ((brakePosition / 32767) * HEAT_UP_FACTOR) * 2;` (remove second star)

Reply:
Why? Unless squaring it wasn't intended. The brackets would be be useless as well if it wasn't supposed to be squared

Reply:
@Meqolo, currently brake temperature is an exponential function of brake position as in

https://www.desmos.com/calculator/cfosoddvhh

`f(x, h) = ((x / 32,767) * h) ^ 2`

where `x` is brake position and 
where `h` is a heat up factor used to speed up / slow down heat up if necessary and
where `0 ≤ x ≤ 32,767`

Exponentiation is intended as @lukecologne says.

Reply:
@Benjozork  Just FYI, that's not an exponential function. That's just a polynomial, or in this case a square function (don't know if it's called that in english, not my first language)

Reply:
@lukecologne, oops English is not my first language either and that's what we call it here weirdly. Thanks for the heads up.